src/service/receipt.service.ts

```diff
@@ -0,0 +1,130 @@
+
+import chalk from 'chalk';
+import {
+    ArtifactExpiryReceipt,
+    BalanceChangeReceipt,
+    BalanceTransferReceipt,
+    InflationReceipt,
+} from 'nem2-sdk';
+
+export class ReceiptService {
+    constructor() {
+
+    }
+
+    /**
+     * @description The logic of formatting Receipt info
+     * @param receipt
+     * @returns {string}
+     */
+    public formatReceiptToFilter(statement: any): string {
+        let txt = '';
+
+        // render transactionStatements
+        txt += chalk.green('transactionStatements:\t') + '\n';
+        txt += '-'.repeat('transactionStatements:\t'.length) + '\n\n';
+        for (let transactionIdx = 0; transactionIdx < statement.transactionStatements.length; transactionIdx++) {
```

Comment:
Not a stopper, but I'd prefer if you could use maps instead of for to follow how we iterate arrays in the rest of the project and avoiding mistakes with array positions.